File: RxHttp/src/main/java/com/github/kongpf8848/rxhttp/callback/SimpleHttpCallback.java
package com.github.kongpf8848.rxhttp.callback;

import java.lang.reflect.Type;

public class SimpleHttpCallback<T> extends HttpCallback<T> {

    public SimpleHttpCallback() {
        super();
    }

    public SimpleHttpCallback(Type type) {
        super(type);
    }

    @Override
    public void onStart() {

    }

    @Override
    public void onNext(T response) {

    }

    @Override
    public void onError(Throwable e) {

    }

    @Override
    public void onComplete() {

    }

    @Override
    public void onProgress(long readBytes, long totalBytes) {

    }
}

File: RxHttp/src/main/java/com/github/kongpf8848/rxhttp/callback/SimpleHttpCallback.kt
package com.github.kongpf8848.rxhttp.callback

import java.lang.reflect.Type

open class SimpleHttpCallback<T>(type: Type) : HttpCallback<T>(type) {
    override fun onStart() {}
    override fun onNext(response: T?) {}
    override fun onError(e: Throwable?) {}
    override fun onComplete() {}
    override fun onProgress(readBytes: Long, totalBytes: Long) {}
}